internal/domain/filter.go

```diff
@@ -566,41 +580,19 @@ func (f Filter) checkSizeFilter(r *Release, minSize string, maxSize string) bool
 
 	if r.Size == 0 {
 		r.AdditionalSizeCheckRequired = true
-
 		return true
 	} else {
 		r.AdditionalSizeCheckRequired = false
 	}
 
-	// if r.Size parse filter to bytes and compare
-	// handle both min and max
-	if minSize != "" {
-		// string to bytes
-		minSizeBytes, err := humanize.ParseBytes(minSize)
-		if err != nil {
-			r.addRejectionF("size: invalid minSize set: %s err: %q", minSize, err)
-			return false
-		}
-
-		if r.Size <= minSizeBytes {
-			r.addRejection("size: smaller than min size")
-			return false
-		}
-
+	ok, err := f.ReleaseSizeOkay(r.Size)
+	if err != nil {
+		r.addRejectionF("size: error checking release size against filter: %+v", err)
+		return false
 	}
-
-	if maxSize != "" {
-		// string to bytes
-		maxSizeBytes, err := humanize.ParseBytes(maxSize)
-		if err != nil {
-			r.addRejectionF("size: invalid maxSize set: %s err: %q", maxSize, err)
-			return false
-		}
-
-		if r.Size >= maxSizeBytes {
-			r.addRejection("size: larger than max size")
-			return false
-		}
+	if !ok {
+		r.addRejection("size: release size out of bounds")
```

Comment:
This generic rejection might makes it harder for users to interpret if it was below or above their filters. With these current changes we don't expose the parsed min,max bytes. Thoughts?

Reply:
So, I think the interface is cleanest as it was originally. It does hide the rejection details though. 

What do you think about this new version where we transmit some details up in an `error`? I'm not in love with tightly coupling the error with the text of the rejection reason like I did. 

Another option I considered was to return the too small / okay / too big info in an `int` a la https://pkg.go.dev/math/big#Float.Cmp or something, then letting the caller build the error message using the `f.MaxSize` string / even expose the parsing method and use that. 

IDK WDYT

Reply:
Much better now! Thank you!